src/crimson/net/SocketMessenger.cc

```diff
@@ -165,9 +165,30 @@ SocketMessenger::bind(const entity_addrvec_t& addrs)
                           [this, addrs] (auto& count) {
     return seastar::repeat_until_value([this, addrs, &count] {
       assert(count >= 0);
+
+      int ms_bind_port_min=0;
+      int ms_bind_port_max=0;
+
```

Comment:
... and this one.

Reply:
Thanks for the kind explanation. Since I'm in a different field, I'm not familiar with writing code in C++ and it's clear that my code wasn't assimilated "especially" for Crimson.

I see that there is a "viable way currently exists" so I close the PR, but the kind explanation is still appreciated.